### PR TITLE
JUnitj v5 more progression

### PR DESCRIPTION
**Pull request by nhojpatrick**
Everything with this pr and PR #242 now migrated to JUnit v5, unless it depends upon junit.framework.TestCase.

### CODE REVIEW

**Comment by garydgregory** [comment]
@nhojpatrick Note that the tests are faiing.

**Comment by nhojpatrick** [comment]
@garydgregory Have removed failing tests as I couldn't quickly fix, will get in the next batch.

File: commons-vfs2/src/test/java/org/apache/commons/vfs2/RunTest.java [hunk 1]
```diff
@@ -23,6 +23,7 @@
 import junit.framework.TestResult;
 
 public class RunTest {
+
```
Comment (garydgregory) [inline commons-vfs2/src/test/java/org/apache/commons/vfs2/RunTest.java:26]: In the i

**Review by garydgregory — APPROVED** [review]
See 1 comment,  no style changes, it's annoying to review 167 files when so many contain ONLY adding or removing blank lines.

**Pull request merged**